docs/reference/setup/install.asciidoc

```diff
@@ -98,6 +98,18 @@ the bundled JVM are treated as if they were within {es} itself.
 The bundled JVM is located within the `jdk` subdirectory of the {es} home
 directory. You may remove this directory if using your own JVM.
 
+[discrete]
+[[jdk-locale]]
+=== JDK locale database
+
+The locale database used by {es}, used to map from various date formats to
+the underlying date storage format, depends on the version of the JDK
+that {es} is running on. On JDK version 23 and above, {es} will use the
+_CLDR_ database. On JDK version 22 and below, {es} will use the _COMPAT_
+database. This may mean that the strings used for textual date formats,
+and the output of custom week-date formats, may change when moving from
+a previous JDK version to JDK 23 or above.
```

Comment:
Would it be appropriate to advise people specifically to avoid this? Specific things to avoid maybe?

Reply:
Added some info